SECP-service/src/main/java/com/visucius/secp/models/Group.java

```diff
@@ -0,0 +1,69 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "group")
+public class Group {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id", unique = true, nullable = false)
+    private int id;
+
+    @Column(name = "name", unique = true, nullable = false)
+    private String name;
+
+    @Column(name = "visibility", nullable = false)
+    private String visibility;
```

Comment:
Why are you making the visibility a string?? I thought the visibility of a group is based on roles. Shouldn't there be a relation between the group and roles table. Just like how a user can have roles. A group also have roles.